### DISCUSSION TITLE
not able to move past step1

### DISCUSSION THREAD

**Discussion by Anjusha7** [skills/.github#313] [Introduction to GitHub]
can someone help me move past the first step after creating a branch from the main using "my-first-brach". after refreshing the next steps are not visible and hence can't finish the module. any help is greatly appreciated.

**Comment by Anjusha7** [comment]
some technical issue. got updated.

**Reply by OKGaosenkwe** [reply]
Did you manage to move to the next step?

**Comment by sinsukehlab** [comment]
It seems that you haven't created a new branch. You can't create a new `main` branch or any branch with a name of an existing branch. Create a `my-first-branch` branch.

_Originally posted by @sinsukehlab in https://github.com/skills/.github/discussions/284#discussioncomment-6553700_

**Reply by Data-Engineer-Joe** [reply]
ok it was not clear enough, that is has to be exactly the name "my-first-branch" I took a random name and nothing happend. 🍡

**Reply by Shahmeer-Nasir** [reply]
Same thing happened to me also.

**Comment by tc-git-1** [comment]
I encountered this problem.
I had created a new branch with a name other than "my-first-branch".
I renamed the branch I created to "my-first-branch", and the problem was resolved.  
Now, when I go to https://github.com/thomasical/skills-introduction-to-github/blob/my-first-branch/README.md , I see:

"Introduction to GitHub
Get started using GitHub in less than an hour.

Step 2: Commit a file ..."

**Comment by md-abid-hussain** [comment]
How i can restart this assignment

**Reply by sinsukehlab** [reply]
It seems that you have generated a repository from education/Series-Intro-to-GitHub-Flow with GitHub Classroom.
You can generate another repository from skills/introduction-to-github.
